index.js

```diff
@@ -262,8 +262,10 @@ alexa.session = function(session) {
     this.details = {
```

Comment:
Hey, I'm stuck with the warning part, maybe you have a suggestion to solve this? I looked up a lot of stuff but still can't find what I need.

I solved the backwards compatibility like that (is that a proper way?):
```
  // Load whole session since 4.0.1 
    this.details = session;
    // Adding userdata again to be Backward compatible
    this.details.userId = session.user.userId; 
    this.details.accessToken = session.user.accessToken;
```

My idea to put in a warning was something like that:
```
  // Load whole session since 4.0.1 
    this.details = session;
    // Adding userdata again to be Backward compatible
    var olduserId = function() {
          console.error('WARNING: The syntax for userId has changed, please check alexa-app/UPGRADING.md for details.');
          return session.user.userId;
    };
    this.details.userId = olduserId();`
```
But now it gives out warning everytime someone uses the details object. It would be better to give a warning only when someone uses details.userId or details.accessToken.

Do you have an idea what I can do, or is there a name for that kind of warning, so I can search for a solution?

Thanks